book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] When adding a rental, we're passing the date to the add_rental function. Kindly modify this method to receive a date when adding a rental. 
You could work it out in this form below 👍🏽
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```